gradle/libs.versions.toml

```
@@ -1,8 +1,8 @@
 [versions]
-kotlin = "1.9.23"
+kotlin = "2.1.10"
 detekt = "1.23.5"
-hilt = "2.51"
-agp = "8.3.2"
+hilt = "2.56.2"
+agp = "8.9.2"
```

```suggestion
agp = "8.10.0"
```

ただ、これをする場合は以下も考慮すべきです。

```shell-session
➜  twinte-android git:(update-project) ✗ ./gradlew detekt

> Configure project :app
WARNING: The option setting 'android.defaults.buildfeatures.buildconfig=true' is deprecated.
The current default is 'false'.
It will be removed in version 10.0 of the Android Gradle plugin.
To keep using this feature, add the following to your module-level build.gradle files:
    android.buildFeatures.buildConfig = true
or from Android Studio, click: `Refactor` > `Migrate BuildConfig to Gradle Build Files`.
```